db/migrations/20181210092417-createEmailAutomationTable.js

```diff
@@ -0,0 +1,51 @@
+module.exports = {
+  up: (queryInterface, DataTypes) => queryInterface.createTable('emailAutomation', {
```

Comment:
@IAMOTZ , don't we need to save the email content as well?

Reply:
Yp, we need to save the email content.
That would even make it easier to resend the email if it fails, I would update the model.